Task-4-blockchain-node/modules/remote_state/templates/remote-state.tpl

```diff
@@ -0,0 +1,7 @@
+terraform {
```

Comment:
We doo not need this

Reply:
@praizz just the template or the whole remote state module?